docs_src/src/pages/documentation/example_app/redirection.mdx

```diff
@@ -0,0 +1,31 @@
+export const description =
+  "Welcome to the Robyn API documentation. You will find comprehensive guides and documentation to help you start working with Robyn as quickly as possible, as well as support if you get stuck.";
+
+## Redirection
+
+Batman wanted to redirect some endpoints to others. Robyn helped him do so by the following.
+
+```python {{ title: 'Redirection' }}
+from robyn import Robyn
+
+app = Robyn(__file__)
+
+@app.get("/")
+def index():
+    return landing()
```

Comment:
Hey @VishnuSanal , this is not a correct way to redirect.

This should be the way

```
@app.get("/307")
async def redirect():
    return Response(
        status_code=307,
        description="",
        headers={"Location": "redirect_route"},
    )

```